app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    hidden = False
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which is correct for tracking all instances. However, ensure that objects are properly removed from this list when they are no longer alive to prevent memory leaks or incorrect length calculations.